main.c

```diff
@@ -246,6 +260,8 @@ int editorReadKey() {
 		} else if (seq[0] == CTRL('x')) {
 			return SWAP_MARK;
 		} else if (seq[0] == 'b' || seq[0] == 'B' || seq[0] == CTRL('b')) {
+		} else if (seq[0] == 'h') {
+            return MARK_BUFFER;
 			return SWITCH_BUFFER;
```

Comment:
Think this is meant to be
```suggestion
		} else if (seq[0] == 'b' || seq[0] == 'B' || seq[0] == CTRL('b')) {
			return SWITCH_BUFFER;
		} else if (seq[0] == 'h') {
			return MARK_BUFFER;
```